libctru/include/3ds/svc.h

```diff
@@ -994,7 +994,7 @@ void svcBreak(UserBreakType breakReason);
  * @param croInfo Library information.
  * @param croInfoSize Size of the above structure.
  */
-void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize);
+void svcBreakRO(UserBreakType breakReason, const void* croInfo, u32 croInfoSize) __asm__("svcBreak");
```

Comment:
Not sure it'll be beneficial: current homebrew and official software use the "default" version of svcBreak with one argument...

The non-default version of svcBreak is only used for debugging purposes.